armi/reactor/blocks.py

```diff
@@ -18,29 +18,31 @@
 
 Assemblies are made of blocks. Blocks are made of components.
 """
-from typing import Optional, Type, Tuple, ClassVar
 import collections
 import copy
 import math
+from typing import ClassVar, Optional, Tuple, Type
 
 import numpy as np
 
-from armi import nuclideBases
-from armi import runLog
+from armi import nuclideBases, runLog
 from armi.bookkeeping import report
 from armi.nucDirectory import elements
 from armi.nuclearDataIO import xsCollections
-from armi.physics.neutronics import GAMMA
-from armi.physics.neutronics import NEUTRON
-from armi.reactor import blockParameters
-from armi.reactor import components
-from armi.reactor import composites
-from armi.reactor import geometry
-from armi.reactor import grids
-from armi.reactor import parameters
-from armi.reactor.components import basicShapes
-from armi.reactor.components.basicShapes import Hexagon, Circle
-from armi.reactor.components.complexShapes import Helix
+from armi.physics.neutronics import GAMMA, NEUTRON
+from armi.reactor import (
+    blockParameters,
+    components,
+    composites,
+    geometry,
+    grids,
+    parameters,
+)
+from armi.reactor.components.basicShapes import Circle, Hexagon, Rectangle
+from armi.reactor.components.complexShapes import Helix, HoledHexagon
+
+# this unused import is required so that the RadialSegment component type is registered
+from armi.reactor.components.volumetricShapes import RadialSegment
```

Comment:
Does this mean radial segments are not nominally registered for use in blueprints?